code/model/Learner.cpp

```diff
@@ -745,16 +878,20 @@ void SubgoalLearner::Iterate (int _iIteration, bool _bTestMode)
 	{
 		int iIndex = ite->first;
 		SubgoalSequenceState& rState = ite->second;
-		Subgoal* pSubgoal = rState.p_Sequence->GetSubgoal (1);
-    if (false == pSubgoal->b_isQuestion) {
-      o_FFInterface.SendTask (iIndex,
-                              i_DomainPddlId,
-                              pSubgoal->s_ProblemPddl,
-                              i_CurrentFFTimelimit);
-    } else {
-      // Do not send the question
-      set_PendingSequences.erase (iIndex);
```

Comment:
All problems are sent as before

We just move along the line to first subgoal for each problem